user-service/src/main/java/org/java/mentorship/user/crypt/MD5.java

```diff
@@ -0,0 +1,13 @@
+package org.java.mentorship.user.crypt;
+
+import lombok.NoArgsConstructor;
+import org.springframework.util.DigestUtils;
+
+@NoArgsConstructor
```

Comment:
Lombok has a @Utility annotation that transforms this class into an utility class

final, static, private constructor I think

Reply:
It's experimental, will try it